packages/app/pages/index.tsx

```diff
@@ -11,19 +11,33 @@ import React, {
   useState,
 } from "react";
 import Web3 from "web3";
-import { Contract } from "web3-eth-contract";
+import { Contract, ContractSendMethod } from "web3-eth-contract";
 import { TransactionReceipt } from "web3-core";
-import { AbiType, StateMutabilityType } from "web3-utils";
+import { AbiType, StateMutabilityType, AbiItem } from "web3-utils";
 import { Modal, Input, Button } from "@cabindao/topo";
 import { styled } from "../stitches.config";
 import BN from "bn.js";
 import passportFactoryJson from "../../contracts/artifacts/contracts/PassportFactory.sol/PassportFactory.json";
+import passportJson from "../../contracts/artifacts/contracts/Passport.sol/Passport.json";
 
 const contractAddressesByNetworkId: {
   [id: number]: { passportFactory: string };
 } = {
-  0x7a69: { passportFactory: "0x5FbDB2315678afecb367f032d93F642f64180aa3" },
+  0x7a69: { passportFactory: "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9" },
```

Comment:
yea we need a better solution for this. Might just use a NEXT_PUBLIC_ENV variable, though even that has some drawbacks